administrator/components/com_joomgallery/helpers/config.php

```diff
@@ -400,6 +404,9 @@ public function __construct($id = null)
         JError::raiseError(500, JText::_('Error loading config data'));
       }
 
+      JPluginHelper::importPlugin('content');
+      $this->_mainframe->triggerEvent('onContentPrepareData', array(_JOOM_OPTION.'.config', (object)$properties));
```

Comment:
$this->_mainframe is not declared in this content, this call will throw an error